apps/bot/src/api/app.module.ts

```diff
@@ -8,11 +8,12 @@ import { AppService } from "./app.service";
 @Module({
   imports: [
     TypeOrmModule.forRootAsync({
-      useFactory: async () => ({
-        ...(await getConnectionOptions()),
-        name: "microservice",
-        entities: Object.values(typeOrmEntities),
-      }),
+      useFactory: async () =>
+        ({
+          ...(await getConnectionOptions()),
+          name: "microservice",
+          entities: Object.values(typeOrmEntities),
+        } as any),
```

Comment:
Need to check does bot works correctly after workround.

Reply:
Hello. I can make it, but not now.

Reply:
Thank you but it's more like self todo.

Reply:
Working ok.